libpod/runtime_ctr.go

```diff
@@ -564,3 +566,16 @@ func (r *Runtime) Export(name string, path string) error {
 	return ctr.Export(path)
 
 }
+
+// RemoveContainersFromStorage attempt to remove containers from storage that do not exist in libpod database
+func (r *Runtime) RemoveContainersFromStorage(ctrs []string) {
+	for _, i := range ctrs {
+		// if the container does not exist in database, attempt to remove it from storage
+		if _, err := r.LookupContainer(i); err != nil && errors.Cause(err) == image.ErrNoSuchCtr {
+			r.storageService.UnmountContainerImage(i, true)
+			if err := r.storageService.DeleteContainer(i); err != nil && errors.Cause(err) != storage.ErrContainerUnknown {
```

Comment:
I think returning ErrContainerUnknown is probably better than dropping it - useful to know if we did nothing in these cases, I think.

Reply:
I think this is going to be a common case, since we might have attempted to remove 5 containers, and 4 succeeded and one failed on the normal path.  Then running through this path would complain about the 4 successful ones.  

I think it is better to just ignore containers that are already removed.  Similar to rm -f /etc/foobar, will not care if /etc/foobar exists.